Triangulation/doc/Triangulation/CGAL/TDS_full_cell_default_storage_policy.h

```diff
@@ -0,0 +1,10 @@
+
+namespace CGAL {
+
+  /*!
+  \ingroup PkgTriangulationsRef
+  A tag class to indicate that mirror indices are not stored in full cells.
+  */
+class TDS_full_cell_mirror_storage_policy
```

Comment:
copy-pasto?

(we may want to introduce a new name/alias for this one, especially if we end up using "mirror" by default)